### PR TITLE
Added Thrift (HBase management in C++)

### PR DESCRIPTION
**Pull request by alfredo-gimenez**

### CODE REVIEW

File: var/spack/packages/thrift/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    version('0.9.2', '89f63cc4d0100912f4a1f8a9dee63678')
+
+    depends_on("autoconf")
+    depends_on("automake")
+    depends_on("bison")
+    depends_on("boost")
+    depends_on("flex")
+    depends_on("jdk")
+    depends_on("libtool")
+    depends_on("lua")
```
Comment (tgamblin) [inline var/spack/packages/thrift/package.py]: This says it depends on lua, but it's built `--with-lua=no` below.  We could make this an optional dependency, but before you do that you need to commit the `lua` package.  Add another commit to the request?

**Comment by alfredo-gimenez** [comment]
Right! Lua was giving me problems, I removed the dep.

File: var/spack/packages/thrift/package.py (hunk 1 above)
Comment (alfredo-gimenez) [inline var/spack/packages/thrift/package.py]: Right! Lua was giving me problems, I removed the dep.

**Comment by tgamblin** [comment]
I tweaked the commits a bit to remove the superfluous merge.  This is integrated into develop now.

**Pull request closed without merging**